### PR TITLE
track view isolated to another file

### PR DESCRIPTION
**Pull request by rafalkowalski**
I'd like to isolate rest of views, it allows us to maintain code easier
@leto @aditya1702 Please review

### CODE REVIEW

File: open_event/views/admin/track_view.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        return self.render('admin/model/create_model.html',
+                           form=form,
+                           event_id=event_id,
+                           events=events,
+                           cancel_url=url_for('.index_view', event_id=event_id))
+
+    @expose('/delete', methods=('GET', 'POST'))
+    def delete_view(self, event_id):
+        track_id = request.args.get('track_id', '')
+        """Delete track method"""
```
Comment (juslee) [inline open_event/views/admin/track_view.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/triplez/open-event-orga-server/file/2530490957/issues/source?bid=3233864&fileBranchId=3283317#l67)

File: open_event/views/admin/track_view.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                           cancel_url=url_for('.index_view', event_id=event_id))
+
+    @expose('/edit', methods=('GET', 'POST'))
+    def edit_view(self, event_id):
+        """Edit track view"""
+        track_id = request.args.get('track_id', '')
+        track = DataGetter.get_object(Track, track_id)
+        events = DataGetter.get_all_events()
+        form = TrackForm(obj=track)
+        self.name = "Track | Edit"
```
Comment (juslee) [inline open_event/views/admin/track_view.py]: Codacy Issue found: [Attribute 'name' defined outside __init__](https://www.codacy.com/app/triplez/open-event-orga-server/file/2530490957/issues/source?bid=3233864&fileBranchId=3283317#l50)

File: open_event/views/admin/track_view.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                flash("You don't have permission!")
+            return redirect(url_for('.index_view', event_id=event_id))
+        return self.render('admin/model/create_model.html',
+                           form=form,
+                           event_id=event_id,
+                           events=events,
+                           cancel_url=url_for('.index_view', event_id=event_id))
+
+    @expose('/delete', methods=('GET', 'POST'))
+    def delete_view(self, event_id):
```
Comment (juslee) [inline open_event/views/admin/track_view.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2530490957/issues/source?bid=3233864&fileBranchId=3283317#l65)

File: open_event/views/admin/track_view.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                           objects=tracks,
+                           event_id=event_id,
+                           events=events)
+
+    @expose('/new', methods=('GET', 'POST'))
+    def create_view(self, event_id):
+        """New track view"""
+        events = DataGetter.get_all_events()
+        form = TrackForm(request.form)
+        self.name = " Track | New"
```
Comment (juslee) [inline open_event/views/admin/track_view.py]: Codacy Issue found: [Attribute 'name' defined outside __init__](https://www.codacy.com/app/triplez/open-event-orga-server/file/2530490957/issues/source?bid=3233864&fileBranchId=3283317#l29)

File: open_event/views/admin/track_view.py [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+from flask.ext import login
+
+
+class TrackView(ModelView):
+    @expose('/')
+    def index_view(self, event_id):
+        """Track list view"""
+        tracks = DataGetter.get_tracks(event_id)
+        events = DataGetter.get_all_events()
+        self.name = "Track"
```
Comment (juslee) [inline open_event/views/admin/track_view.py]: Codacy Issue found: [Attribute 'name' defined outside __init__](https://www.codacy.com/app/triplez/open-event-orga-server/file/2530490957/issues/source?bid=3233864&fileBranchId=3283317#l18)

File: open_event/views/admin/track_view.py [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+        return self.render('admin/model/create_model.html',
+                           form=form,
+                           event_id=event_id,
+                           events=events,
+                           cancel_url=url_for('.index_view', event_id=event_id))
+
+    @expose('/delete', methods=('GET', 'POST'))
+    def delete_view(self, event_id):
+        track_id = request.args.get('track_id', '')
+        """Delete track method"""
```
Comment (juslee) [inline open_event/views/admin/track_view.py]: Codacy Issue found: [String statement has no effect](https://www.codacy.com/app/triplez/open-event-orga-server/file/2530633423/issues/source?bid=3233864&fileBranchId=3283317#l65)

File: open_event/views/admin/track_view.py [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+                flash("You don't have permission!")
+            return redirect(url_for('.index_view', event_id=event_id))
+        return self.render('admin/model/create_model.html',
+                           form=form,
+                           event_id=event_id,
+                           events=events,
+                           cancel_url=url_for('.index_view', event_id=event_id))
+
+    @expose('/delete', methods=('GET', 'POST'))
+    def delete_view(self, event_id):
```
Comment (juslee) [inline open_event/views/admin/track_view.py]: Codacy Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2530633423/issues/source?bid=3233864&fileBranchId=3283317#l63)

**Comment by aditya1702** [comment]
@rafalkowalski I also like the idea of having different views for each model. :+1:

**Pull request merged**